go/oasis-node/cmd/common/common.go

```diff
@@ -57,6 +59,21 @@ func DataDirOrPwd() (string, error) {
 	return dataDir, nil
 }
 
+func EntityFactory(signerBackend string, entityDir string) (signature.SignerFactory, error) {
```

Comment:
The method is strangely named and is missing a documentation block. It should be something like `SignerFactory` instead as it has nothing to do with an entity.

Also why do you use `signerBackend` and `entityDir` (which should be `fileSignerDir`) and not just use the `SignerDirOrPwd` and other flags instead?

Reply:
Ah it should be `SignerFactory` and `signerDir`. I renamed `fileSignerDir` to `signerDir` because it seems that signers will require a directory no matter what, not just for signers with a file backend.